content/rs/administering/active-passive.md

```diff
@@ -38,7 +38,7 @@ result, commands that were executed in a certain order when compared
 across source databases might be executed in a different order on the
 destination database.
 
-**Note:** The *Replica of* feature should not be confused with the
+**Note:** The Replica Of feature should not be confused with the
```

Comment:
Why not keep "Replica Of" in italics?

Reply:
We generally refer to feature names in caps and without italics. Do you think we should change that?